contrib/helm/harbor/values.yaml

```diff
@@ -280,3 +275,13 @@ clair:
 #
 notary:
   enabled: false
+
+## Settings for postgresql dependency.
+## see https://github.com/kubernetes/charts/tree/master/stable/postgresql
+## for further configurables.
+postgresql:
+  postgresUser: clair
+  postgresPassword: not-a-secure-password
```

Comment:
Thanks Paul for this patch!  Could we link clair.postgresPassword to postgresql.postgresPassword in values.yaml because they must be the same ?

Reply:
I specifically didn't do that because there's a good chance harbor is changing its database to postgres too and we probably want clair and harbor to have different database/username/password to postgres.

I can document them better in the values.yaml file to make it more clear that they need to match.

Reply:
Didn't catch your point on "we probably want clair and harbor to have different database/username/password to postgres." 
Code is always better than doc when the code doesn't take big extra effort.
The clair database username/password/schema must match the username/password/schema specified for postgresql chart, then we'd better reference the former to latter. Is there other reason you don't prefer using 'password: &clair_db_password xxx' in values.yaml ?
```
clair:
  enabled: true
  image:
    repository: vmware/clair-photon
    tag: v2.0.1-v1.4.0
    pullPolicy: IfNotPresent
  postgresPassword: not-a-secure-password
  postgresUser: clair
postgresDatabase: clair
```

Reply:
We'd better not expect the end users to read the doc in clair section when he modifies password in postgresql section.

Reply:
have you got docs on using 'password: &clair_db_password xxx' ?  I haven't seen it in other charts and want to make sure I understand it fully before I use it.

Reply:
I may also update it to be better at providing options for external database vs the depdendency,  something like the way wordpress does it ( https://github.com/kubernetes/charts/blob/master/stable/wordpress/values.yaml ) combined with your pointer trick above (I may chat with helm community about that first and make sure its a viable path for helm charts in general since  i don't see it in existing charts I've looked at )

Reply:
ahhh YAML anchors.  I learn something today :)

Reply:
Spoke to a few of the core helm chart maintainers, and they suggest we should avoid YAML anchors as they're only processed once when values.yaml are read and are not re-evaluated when using `--set` and `--values` to override.  I'll update this to look more like how the stable/wordpress chart handles external DB.